homeassistant/components/lock/zwave.py

```diff
@@ -228,21 +239,29 @@ def update_properties(self):
         _LOGGER.debug("Lock state set from Bool value and is %s", self._state)
         if self.values.access_control:
             notification_data = self.values.access_control.data
-            self._notification = LOCK_NOTIFICATION.get(str(notification_data))
+            self._notification = LOCK_NOTIFICATION.get(
+                str(notification_data))
```

Comment:
Preference: I know it was already like this, but can we make the dict keys integers and remove this string cast, or is there something I'm missing here?